doc/man3/SSL_CTX_set_ssl_version.pod

```diff
@@ -35,6 +37,11 @@ When L<SSL_clear(3)> is called and no session is connected to
 an SSL object, the method of the SSL object is reset to the method currently
 set in the corresponding SSL_CTX object.
 
+SSL_CTX_set_version() has unusual semantics and no clear use case;
+it would usually be preferable to create a new SSL_CTX object than to
+try to reuse an existing one in this fashion.  Its usage should be
+considered deprecated.
+
```

Comment:
Please add the explicit deprecation notice into a new HISTORY section similarly as in other deprecated functions.